Orange/widgets/data/owrank.py

```diff
@@ -488,6 +494,17 @@ def selectedAttrs(self):
         else:
             return []
 
+    def create_scores_table(self):
+        features = [ContinuousVariable(s[0]) for s in self.measures]
+        metas = [StringVariable("Feature names")]
+        domain = Domain(features, metas=metas)
+
+        scores = np.array(self.measure_scores).T
+        feature_names = np.array([a.name for a in self.data.domain.attributes])
+        # Reshape to 2d array as Table does not like 1d arrays
+        feature_names = feature_names[:, None]
```

Comment:
Creates a row instead of a column, and I dislike the .T. The optimal way would be for Table to do this transformation instead.